src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -1,22 +1,566 @@
+      handleOnAssetAdded(DAO_LOGO_MOBILE, base64, fileExtension)
+    },
+    [handleOnAssetAdded]
+  )
+
+  const handleOnTokenLogoLoaded = useCallback(
+    (base64, fileExtension) => {
+      handleOnAssetAdded(TOKEN_LOGO, base64, fileExtension)
+    },
+    [handleOnAssetAdded]
+  )
+
+  const handleOnAssetRemoved = useCallback(type => {
+    setFormData(formData => {
+      return {
+        ...formData,
+        [type]: '',
+      }
+    })
+  }, [])
+
+  const handleOnDaoLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(DAO_LOGO)
+  }, [handleOnAssetRemoved])
+
+  const handleOnMobileLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(DAO_LOGO_MOBILE)
+  }, [handleOnAssetRemoved])
+
+  const handleOnTokenLogoRemoved = useCallback(() => {
+    handleOnAssetRemoved(TOKEN_LOGO)
+  }, [handleOnAssetRemoved])
+
+  const handleNext = useCallback(() => {
+    onConfigChange('garden', formData)
+    onNext()
+  }, [onConfigChange, onNext, formData])
 
   return (
     <div>
-      {title}
+      <div
+        css={`
+          margin-bottom: ${6 * GU}px;
+          text-align: center;
+        `}
+      >
+        <h1
+          css={`
+            ${textStyle('title1')};
+            margin-bottom: ${3 * GU}px;
+          `}
+        >
+          Garden Metadata
+        </h1>
+        <div>
+          <p
+            css={`
+              ${textStyle('body1')};
+              color: ${theme.contentSecondary};
+            `}
+          >
+            Fill with your garden information
+          </p>
+        </div>
+      </div>
+      <div
+        css={`
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          margin-bottom: ${6 * GU}px;
+        `}
+      >
+        <Field
+          label="Garden Name"
+          css={`
+            width: 100%;
+            marging-bottom: ${2 * GU}px;
+          `}
+        >
+          <TextInput
+            css="width: 100%;"
+            maxLength="15"
+            onChange={handleGardenNameChange}
+            value={formData.name}
+            required
+          />
+        </Field>
+        <Field
+          label="Garden Description"
+          css={`
+            width: 100%;
+            marging-bottom: ${2 * GU}px;
+          `}
+        >
+          <TextInput
+            multiline
+            maxLength="120"
+            css="width: 100%;"
+            onChange={handleGardenDescriptionChange}
+            value={formData.description}
+            required
+          />
+        </Field>
+        <Field
+          label="Assets"
+          css={`
+            width: 100%;
+          `}
+        >
+          <Box
+            css={`
+              width: 100%;
+            `}
+          >
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+                justify-content: space-around;
+              `}
+            >
+              <Field label="Dao Logo">
```

Comment:
Was thinking if we should update to garden everywhere in the UI?
I think this is the first time we display Dao nomenclature 
```suggestion
              <Field label="Garden Logo">
```